### ISSUE TITLE
Is there a way to redefine the trigger key?

### ISSUE THREAD

**Issue by Unbrick** [bobbylight/RSTALanguageSupport#59]
Hey there,

thank you for the awsome project. Sadly currently i'm facing the problem that the trigger key is already in use and therefore the autocompletion wont come up.

Is there any way to redefine the trigger key?

Thank you in advance!

**Comment by Hasnatrasool163** [comment]
yes you can do in abstract class or pass trigger key to installImp method , so all textareas could trigger autocompletion for user defined KeyStroke : this method would be update to pass a trigger key or use default 

/**
	 * Registers an auto-completion instance.  This should be called by
	 * subclasses in their
	 * {@link #install(org.fife.ui.rsyntaxtextarea.RSyntaxTextArea)} methods
	 * so that this language support can update all of them at once.
	 *
	 * @param textArea The text area that just installed the auto-completion.
//	 * @param ac The auto-completion instance.
	 * @see #uninstallImpl(RSyntaxTextArea)
	 */
	protected void installImpl(RSyntaxTextArea textArea,AutoCompletion ac) {
		textAreaToAutoCompletion.put(textArea, ac);
		ac.setTriggerKey(getKeyStroke());
	}

its part of class 
/**
 * A base class for language support implementations.
 *
 * @author Robert Futrell
 * @version 1.0
 */
public abstract class AbstractLanguageSupport implements LanguageSupport {

**Comment by bobbylight** [comment]
I'm closing this as it was answered by @Hasnatrasool163 - you just need to call [AutoCompletion.setTriggerKey(KeyStroke)](https://github.com/bobbylight/AutoComplete/blob/72408f54b302f53f053d0fbda7266fe78855920b/AutoComplete/src/main/java/org/fife/ui/autocomplete/AutoCompletion.java#L1173) somewhere in the UI setup flow.
